zokrates_js/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zokrates_js"
-version = "1.0.22"
+version = "1.0.23"
```

Comment:
We usually bump version numbers when releasing, as if another PR changes zokrates_js we would end up bumping twice. Is there another reason to bump here?

Reply:
we pushed latest version manually, so 1.0.23 is the latest version on npm, we still need to bump the version for new release to 1.0.24